lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -407,21 +412,25 @@ sub check_parameters {
 
   my @required_fields = qw/referenceName referenceBases assemblyId/;
 
-  if($parameters->{'alternateBases'}){
+  if($parameters->{'alternateBases'}) {
     push(@required_fields, 'alternateBases');
     push(@required_fields, 'start');
```

Comment:
`start` is not required if `alternateBases` is used in a `geneId` query: http://docs.genomebeacons.org/variant-queries/#beacon-geneid-queries

Reply:
That's true but at the moment we don't have a geneId query. This type of query is more complex to implement, something to consider for the future.